components/telemetry-operator/README.md

```diff
@@ -88,15 +88,7 @@ make push-image DOCKER_PUSH_DIRECTORY=<my container repo> DOCKER_TAG=latest
 kubectl -n kyma-system set image deployment telemetry-operator manager=<my container repo>/telemetry-operator:latest
 ```
 
-### Enable Tracing Controller
-
-To activate configurable tracing, follow the previous steps to run the telemetry operator in your development environment. When deploying the operator with the [telemetry chart](https://github.com/kyma-project/kyma/tree/main/resources/telemetry), you must install the `TracePipeline` CRD manually and set a feature flag to enable tracing:
-
-```bash
-kubectl apply -f https://raw.githubusercontent.com/kyma-project/kyma/main/components/telemetry-operator/config/crd/bases/telemetry.kyma-project.io_tracepipelines.yaml
-kyma deploy -s main --value telemetry.operator.controllers.tracing.enabled=true
-```
-
+## Troubleshooting
 
 ### Enable pausing reconciliations
 You must pause reconciliations to be able to debug the pipelines and, for example, try out a different pipeline configuration or a different OTel configuration. To pause reconciliations, create a `telemetry-override-config` in the `kyma-system`
```

Comment:
```suggestion
You must pause reconciliations to be able to debug the pipelines and, for example, try out a different pipeline configuration or a different OTel configuration. To pause reconciliations, create a `telemetry-override-config` in the `kyma-system` Namespace.
```

Let's not split sentences across two lines ;)

Please remove "Namespace. " from line 95; I couldn't make a suggestion for that.